Library/Formula/mono.rb

```diff
@@ -72,6 +89,39 @@ def install
       </Project>
     EOS
     shell_output "#{bin}/xbuild test.csproj"
+
+    # Test that fsharpi is working
+    ENV.prepend_path "PATH", bin
+    output = shell_output %(echo 'printfn "#{test_str}"; exit 0' | #{bin}/fsharpi)
+    assert output.include? test_str
```

Comment:
Can we use an `assert_match` here? We're trying to phase out `output.include?` because the failure message isn't very clear or helpful. Should work as something like:

```ruby
assert_match test_str, output
```
?